single-campaigns.php

```diff
@@ -1,46 +1,108 @@
 <?php
 /**
- * The Template for displaying all single campaign posts
+ * Template Variables for Campaigns.
```

Comment:
This file seems that it was checked out earlier.
There are many changes in planet-2561 (that contains the 1st PR merged for the campaigns feature).
Please synchronize with the one currently on the branch.

58-105 lines were also removed from this template in that 1st PR. Please remove them from here also